apps/package.json

```diff
@@ -11,14 +11,14 @@
     "lint": "eslint . --ext .js,.jsx",
     "test": "npm run lint && grunt test",
     "test-low-memory": "./test-low-memory.sh",
-    "test:unit": "grunt unitTest",
-    "test:integration": "grunt integrationTest",
+    "test:unit": "grunt unitTest; grunt logBuildTimes",
+    "test:integration": "grunt integrationTest; grunt logBuildTimes",
     "clean": "grunt clean",
-    "build": "MOOC_DEV=1 grunt build",
-    "build:dist": "MOOC_DEV=0 grunt clean build",
-    "start": "MOOC_DEV=1 grunt dev",
-    "start:applab": "npm run start -- --app=applab",
-    "start:gamelab": "npm run start -- --app=gamelab",
+    "build": "MOOC_DEV=1 grunt build; grunt logBuildTimes",
+    "build:dist": "MOOC_DEV=0 grunt clean build; grunt logBuildTimes",
+    "start": "MOOC_DEV=1 grunt dev; grunt logBuildTimes",
+    "start:applab": "MOOC_DEV=1 grunt dev --app=applab; grunt logBuildTimes",
+    "start:gamelab": "MOOC_DEV=1 grunt dev --app=gamelab; grunt logBuildTimes",
```

Comment:
How does this work with the dev server?  I always kill it with an interrupt; does the `logBuildTimes` task run after the interrupt?

(Note: I will attempt to answer my own question by running this locally.  Just thought I'd record the question here.)